platform/working-with-vsp/vsp-collaboration-cycle/vsp-collaboration-cycle.md

```diff
@@ -313,11 +313,21 @@ You bring:
 - Link to your [test plan](https://dsvavsp.testrail.io/index.php?/suites/view/2&group_by=cases:section_id&group_order=asc) in TestRail with [test results](https://dsvavsp.testrail.io/index.php?/runs/view/7&group_by=cases:section_id&group_order=asc) _for QA practice area review_ 
 - Link to your [Coverage for References](https://dsvavsp.testrail.io/index.php?/reports/view/12) and [Summary(Defects)](https://dsvavsp.testrail.io/index.php?/reports/view/14) reports in TestRail _for QA practice area review_
```

Comment:
Might we be specific here, too? I think this is QA again.

Reply:
The QA practice area review is defined at the end of this sentence. I'm going to opt to leave this one as is.